core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -367,7 +369,7 @@ var TopicsAndSkillsDashboardPage = function() {
     await waitFor.elementToBeClickable(
       assignedTopicInput,
       'Assigned topic checkbox takes too long to be clickable');
-    await assignedTopicInput.click();
+    await action.click('Assigned Topic Input', assignedTopicInput);
 
     await waitFor.elementToBeClickable(
       confirmUnassignSkillButton,
```

Comment:
Can you add a waitFor in `this.expectNumberOfTopicsToBe` below? You'll need to put it in an if block so that we only wait if `count` is not 0

Reply:
I think we actually made a mistake when writing this code. I checked with the author, and we should change it to better handle the case where count is 0. Here's the approach they suggested:

> When there are no skills or topics on the server, we don't show the table. The UI will just show two buttons on the page.

> I think a better way would be to use waitFor for the table when the number > 0 and if number is 0, expect that the table is not present.

You can see our conversation [here](https://github.com/oppia/oppia/pull/10782/files#r566631072).

Reply:
Done, please excuse my commits, sorry for that.
@U8NWXD